src/test/java/core/basesyntax/service/impl/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    private static PurchaseOperationHandler purchaseOperationHandler;
+
+    @Before
+    public void setUp() {
+        purchaseOperationHandler = new PurchaseOperationHandler();
+    }
+
+    @After
+    public void afterEachTest() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    public void handle_purchaseIsLessThenBalance_notOk() {
+        FruitTransaction fruitTransaction = new FruitTransaction("p", "banana", 81);
+        try {
+            purchaseOperationHandler.handle(fruitTransaction);
+        } catch (RuntimeException e) {
+            return;
+        }
+        fail("The purchase cannot be completed");
+    }
+
+    @Test
+    public void handle_purchaseIsMoreThenBalance_ok() {
+        Storage.storage.put("banana", 80);
+        FruitTransaction fruitTransaction = new FruitTransaction("p", "banana", 60);
```

Comment:
```suggestion
        Storage.storage.put("banana", 80);
        FruitTransaction fruitTransaction = new FruitTransaction("p", "banana", 60);
```

extract strings to constants